dynamiq/utils/feedback.py

```diff
@@ -0,0 +1,35 @@
+from dynamiq.callbacks.streaming import AsyncStreamingIteratorCallbackHandler, StreamingEventMessage
+from dynamiq.runnables import RunnableConfig
+from dynamiq.types.feedback import FeedbackMethod
+
+
+def send_message(
+    event_message: StreamingEventMessage, config: RunnableConfig, approval_type: FeedbackMethod = FeedbackMethod.STREAM
+) -> None:
+    """Emits message to the queue of AsyncStreamingIteratorCallbackHandler
+
+    Args:
+        message (StreamingEventMessage): Message to send.
+        config (RunnableConfig): RunnableConfig with callbacks (AsyncStreamingIteratorCallbackHandler is required).
+    """
+
+    match approval_type:
+        case FeedbackMethod.CONSOLE:
+            print(event_message.data)
+        case FeedbackMethod.STREAM:
+            sender_callback = next(
+                (
+                    callback
+                    for callback in config.callbacks
+                    if isinstance(callback, AsyncStreamingIteratorCallbackHandler)
```

Comment:
`StreamingIteratorCallbackHandler` also could be a case and other callback (that could be developed by user) that provides `on_node_execute_stream` functionality. I think we need to make a more robust check or skip it at all.